src/staking/unbondingAccount.ts

```diff
@@ -267,15 +268,21 @@ export class UnbondingAccount {
         receiver: rentReceiver,
         stakeAccount: stakeAccount.address,
         stakePool: stakeAccount.stakeAccount.stakePool,
```

Comment:
Now that you're passing the stake pool in you can pull the pubkey from it
```suggestion
        stakePool: stakePool.addresses.stakePool,
```